lib/md040.js

```diff
@@ -2,16 +2,27 @@
 
 "use strict";
 
-const { addErrorContext, filterTokens } = require("../helpers");
+const { addErrorContext, filterTokens, addError } = require("../helpers");
 
 module.exports = {
   "names": [ "MD040", "fenced-code-language" ],
   "description": "Fenced code blocks should have a language specified",
   "tags": [ "code", "language" ],
   "function": function MD040(params, onError) {
+    let allowList = params.config.allow_list;
+    allowList = Array.isArray(allowList) ? allowList : null;
```

Comment:
I think other rules us an empty array as a default https://github.com/DavidAnson/markdownlint/pull/610/files#diff-b9c3f0adb939e1b4f7e9c2e963f2f49f6f09a4a5fd71fd4a7a4811732d69aff0R408-R412
Think it might make the code parsing simpler

Reply:
Agree, see MD033.

Reply:
I used empty array as a default in my first implementation, but I thought it would mean no language was allowed semantically, that's why I changed it to `null` later. What do you think?

Reply:
https://github.com/DavidAnson/markdownlint/blob/2b2dc27f24a16a2afdf66d850a247f7ce02df6c8/lib/md033.js#L21-L23